flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -661,8 +690,14 @@ static bool msp_receive_byte(struct msp_bridge *m, uint8_t b)
  */
 static int32_t uavoMSPBridgeStart(void)
 {
-	if (!module_enabled)
+	if (!module_enabled) {
+		// give port to telemetry if it doesn't have one
+		// stops board getting stuck in condition where it can't be connected to gcs
+		if(!PIOS_COM_TELEM_RF)
```

Comment:
So I can live with this here.  My only objection to here instead of the Naze32 target is that if the module isn't compiled in you'll be in the same situation (except this won't run).